doc/classes/Transform.xml

```diff
@@ -135,16 +135,17 @@
 			<argument index="0" name="scale" type="Vector3">
 			</argument>
 			<description>
-				Scales the transform by the specified 3D scaling factors.
+				Scales the transform by the given scale factor, using matrix multiplication.
 			</description>
 		</method>
 		<method name="translated">
 			<return type="Transform">
 			</return>
-			<argument index="0" name="ofs" type="Vector3">
+			<argument index="0" name="offset" type="Vector3">
 			</argument>
 			<description>
-				Translates the transform by the specified offset.
+				Translates the transform by the given offset, relative to the transform's basis vectors.
```

Comment:
Hm, but the transform's basis vectors do contain scale, and that's what `translated` ignores, isn't it?

Reply:
No, it acts as `origin += basis_xform(offset)`.